features/account/managing_account/user_profile_validation.feature

```diff
@@ -0,0 +1,42 @@
+@managing_account
+Feature: User account profile validation
+    In order to avoid making mistakes when changing my personal information
+    As a logged user
+    I want to be prevented from changing fields to incorrect values
+
+    Background:
+        Given the store operates on a single channel in "France"
```

Comment:
Agreed, but I suppose ``the store operates on a single channel in "France"`` has hardcoded country :( it should definitely be changed in the nearest future to avoid such a confusion.
BTW: Your comment should start with **SPOILER ALERT**, as maybe somebody is currently watching season 2? 😱